### PR TITLE
Add my messing about

### PR DESCRIPTION
**Pull request by jondea**
Please have a look at this and review

### CODE REVIEW

**Review by MatthiasHeilManchester — COMMENTED** [review]
have a look at my comments

File: README.md [hunk 1]
```diff
@@ -1,2 +1,4 @@
 # tmp_oomph_dont_use
 messing about
+
+JD is also messing about to show MH what a PR looks like.
```
Comment (jondea) [inline README.md:4]: Test?

Comment (MatthiasHeilManchester) [inline README.md:4]: not sure about line 4

**Review by MatthiasHeilManchester — COMMENTED** [review]
bla about line 4

File: README.md (hunk 1 above)
Comment (jondea) [reply README.md:4]: But I really like line 4!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,4 @@
 # tmp_oomph_dont_use
 messing about
```
Comment (MatthiasHeilManchester) [inline README.md:2]: but what about line 2

**Review by MatthiasHeilManchester — COMMENTED** [review]
comment about line 2 added

**Pull request merged**